packages/web/src/scss/components/Accordion/index.html

```diff
@@ -0,0 +1,197 @@
+{{#> layout/plain }}
+
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Default accordion</h2>
+
+  <div class="mb-400">
+
+    <section
+      id="accordionExample"
+      class="Accordion"
+      data-toggle="accordion"
+    >
+
+      <!-- Accordion Item -->
```

Comment:
Do we actually need all the comments here?

Reply:
I added the comments there because the sample is too complex, so I thought the comments would make it easier to navigate.

Reply:
ok